element_deeplabcut/train.py

```diff
@@ -241,15 +233,10 @@ def make(self, key):
             find_full_path(get_dlc_root_data_dir(), fp).as_posix()
             for fp in (VideoSet.File & key).fetch("file_path")
         ]
-        dlc_config["video_sets"] = video_filepaths
-
-        # ---- Write DLC and basefolder yaml (config) files ----
+        dlc_config["video_sets"] = {v: [] for v in video_filepaths}

```

Comment:
Are the values of the keys empty lists?

Reply:
No, actually. I put this in as a placeholder and forgot to revise. What I've seen has been 
```suggestion
        dlc_config["video_sets"] = {v: f"crop: {x1} {x1} {y1} {y2} for v in video_filepaths}
```
But I haven't figure out the best way to find the crop values. Should we allow them to be provided as secondary attributes in `train.VideoSet.File`? I don't think we need to store our `video_filepaths` in the config, so I think we should remove this and load the information from the original, but reset the root dir.

I do something similar in [this workflow function](https://github.com/CBroz1/workflow-deeplabcut/blob/dev/workflow_deeplabcut/load_demo_data.py#L21)

Reply:
Hi @tdincer - in [this commit](https://github.com/CBroz1/element-deeplabcut/commit/271b00355352ae296c91166743c8ba3337405666) I revise how I handle `video_set` to pull from the existing config file. I then save what we store in `VideoSet.File` (which could be anything) under a new key `training_filelist_datajoint` to avoid overwriting the crop settings.

The changes here pass integration tests for model training and model evaluation.